cosmo_tester/framework/util.py

```diff
@@ -665,10 +665,15 @@ def validate_cluster_status_and_agents(manager,
                                        tenant,
                                        expected_brokers_status='OK',
                                        expected_managers_status='OK',
-                                       agent_validation_manager=None):
+                                       agent_validation_manager=None,
+                                       logger=None):
```

Comment:
I think the logger should be mandatory- it makes it more clear where failures occur when you have preceding lines saying what was being attempted.

Reply:
Agreed, I'll add `logger` to all places that use the function.